docs/2.0/docs/patcher/concepts/patches.md

```diff
@@ -1,3 +1,189 @@
 # What is a Patcher Patch?
 
-This page is under construction
+A Patch is a set of instructions executed by Patcher that do code transformations.
+This strategy is especially useful as a way to automate adoption of a breaking change with infrastructure as code, such as Terragrunt, OpenTofu, or Terraform.
+This instruction sheet is delivered by means of a `yaml` file in a specific format:
+
+```yaml title=".patcher/patches/v1.0.0/mypatch/patch.yaml"
+name: "<name-of-patch>"
+description: "<description for patch>"
+author: <your-name-here> <<your-email-address-here>>
+
+# Optional dependencies. Terrapatch is a typical one
+dependencies:
+  - name: terrapatch
+    version: "0.1.0"
+
+# Steps necessary to resolve breaking change
+steps:
+  - name: "<name-of-step>"
+    run: <command-to-run>
+  - name: "<name-of-second-step>"
+    run: <second-command-to-run>
+  # etc
+```
+
+[Check out an example of a patch in the CIS Service Catalog.](https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/c3d5ede211fc3230a7d493ceea43622b337ee88a/.patcher/patches/v0.96.4/switch-to-cis-rds-module/patch.yaml)
+
+## For Module Consumers
+
+As an OpenTofu/Terraform module consumer, modules are imported to launch infrastructure by writing Terragrunt units (`terragrunt.hcl` files) or OpenTofu/Terraform code that calls a module.
+
+It is best practice to reference a specific version of an OpenTofu/Terraform module. 
+Over time, module authors release new versions of the module, and the code that consumes those modules slowly gets out of date. 
+In some cases, the latest update of the underlying modules requires a breaking change to the consuming code, meaning the version can't just be bumped. 
+The consuming code needs to be edited to make use of the new version. 
+In this instance, using a patch with Patcher comes in handy.
+
+Patches can be consumed with either a "push" strategy, when Patcher proactively opens a pull request with the latest update, or a "pull" strategy, when a repo is manually scanned to look at the current state of your infrastructure using the Patcher CLI tool.
+
+Regardless of methodology, the concept remains the same. 
+Patcher will suggest changes to your codebase in order to keep your infrastructure up to date, however you see fit. 
+
+Here is an example of what your PR will look like, after being generated by Patcher:
+
+![Patcher PR Example](/img/patcher/pr-example.png)
+
+### Update Push Strategy
+
+Let's take a look at the experience of the "push" strategy available through our [GitHub action](https://github.com/gruntwork-io/patcher-action). 
+Using the GitHub action, PRs will be opened against your codebase on a cadence specified by you, against environments, versions, etc. specified by you. 
+The intention with this GitHub action is to leave the repo owner in full control of your upgrade cadence. 
+
+<details>
```

Comment:
I worry this will go stale, and is also too much unexplained detail for a concept page.  Maybe better to just remove it and link instead to the promotion workflows guides?

Reply:
just to confirm which noun we're talking about - remove "it" being the full example?